src/lib/holocene/checkbox.svelte

```diff
@@ -8,6 +8,7 @@
   interface $$Props extends HTMLInputAttributes {
     checked?: boolean;
     label?: string;
+    ariaLabel?: string;
```

Comment:
is `aria-label` not already a valid prop because we're extending `HTMLInputAttributes` ?

Reply:
Wow, yes we certainly are 🤦‍♀️ Updated!